colin-api/src/colin_api/models/business.py

```diff
@@ -327,6 +334,7 @@ def find_by_identifier(cls,  # pylint: disable=too-many-statements
                 else convert_to_json_date(business['last_agm_date'])
             business_obj.last_ledger_timestamp = convert_to_json_datetime(business['last_ledger_timestamp'])
             business_obj.status = business['state']
+            business_obj.admin_freeze = business['admin_freeze']
```

Comment:
I assume you expect a string. Convert if you need boolean value

Reply:
business['admin_freeze'] will have True or False value